taxi/remote.py

```diff
@@ -96,7 +96,7 @@ def _login(self):
         else:
             self.logged_in = True
 
-    def send_entries(self, entries, mappings, callback=None):
+    def send_entries(self, entries, mappings, callback=None, skip_today=False):
```

Comment:
Your are probably right. So can I do that:

1. Update ```entries_to_push = timesheet.get_entries(...)``` to return only the entries to push
1. Do no flag local_entries as commented if they are from today
1. Add the today entries to ```timesheet_collection.get_ignored_entries()```

or maybe it's even simpler to work before all that, and just ignored them from the parsing...